public/v2/apps/eclipse-mosquitto.json

```diff
@@ -0,0 +1,60 @@
+{
+    
+"captainVersion": "2",
+"documentation": "Taken from https://github.com/eclipse/mosquitto. Mosquitto is an open source implementation of a server for version 5.0, 3.1.1, and 3.1 of the MQTT protocol. It also includes a C and C++ client library, and the mosquitto_pub and mosquitto_sub utilities for publishing and subscribing.",
+"displayName": "Eclipse Mosquitto - A MQTT Broker",
+"description": "Eclipse Mosquitto is an open source message broker which implements MQTT version 5, 3.1.1 and 3.1",
+
+
+
+"dockerCompose": {
+    "version": "3.3",
+    "services": {
+        "$$cap_appname": {
+
+  
+            "image": "eclipse-mosquitto:$$cap_eclipse_version",
+            "volumes": [
+                "$$cap_appname-eclipse-config:/mosquitto/config/",
+                "$$cap_appname-eclipse-data:/mosquitto/data/",
+                "$$cap_appname-eclipse-log:/mosquitto/log/"
+            ],
+            "notExposeAsWebApp": "true",
+            "ports": [
+                    "$$cap_mqtt_port:1883",
+                    "$$cap_ws_port:9001"],
```

Comment:
These port don't have to be mapped to host ports necessarily if the app is only being used insternally via other apps hosted on the same CapRover instance. 

Let's remove them.